app/services/notification_creator_for_post_service.rb

```diff
@@ -7,13 +7,15 @@ def initialize(post, current_user)
   def call
     if @post.user.admin?
       User.all.each do |user|
-        Notification.create(sender_id: @post.user.id, receiver_id: user.id, action: :post, content: "Admin has just posted an article", post_id: @post.id)
+        if user != User.admin[0]
```

Comment:
this is bad, what if there are more than 2 admins?

Reply:
oh no